core/frontend/src/tile/OPCFormatInterpreter.ts

```diff
@@ -54,9 +54,14 @@ export class OPCFormatInterpreter  {
     worldRange = Range3d.createXYZXYZ(bounds.getMinX(), bounds.getMinY(), bounds.getMinZ(), bounds.getMaxX(), bounds.getMaxY(), bounds.getMaxZ());
     isGeolocated = false;
     const fileCrs = fileReader.getFileCRS();
-    if (fileCrs) {
+    // the CRS 9300 is not defined in the CRS registry database, so we cannot use CRSManager
+    // Check to isGeographicCRS and isProjectedCRS are both going to return false in that case and we will fallback to
+    // use un-georeferenced code path.
+    await CRSManager.ENGINE.prepareForArea(fileCrs, bounds);
+    const isGeographicCRS = CRSManager.ENGINE.isGeographicCRS(fileCrs);
+    const isProjectedCRS = CRSManager.ENGINE.isProjectedCRS(fileCrs);
```

Comment:
Is `isProjectedCRS` a trivial check, or should you avoid checking it if `isGeographicCRS` is `true`?

Reply:
That is trivial.